data/api.js

```diff
@@ -8,7 +8,8 @@ function post (url, emailAddresses, articleId, customMessage) {
 		body: JSON.stringify({
 			contentUUID: articleId,
 			recipients: emailAddresses,
-			customMessage: customMessage
+			customMessage: customMessage,
+			imageUrl: imageUrl
```

Comment:
presumably you're going to work on the backends next? to add in the image to the emails?